### PR TITLE
Include certbot to deploy scripts

### LINKED ISSUE #735: Include certbot to deploy scripts
It would be great to integrate letsencrypt auto generated certificates to Impress but first step will be adding certbot to `/deploy/*.sh` scripts with `EPEL 7` dependency

### PR DESCRIPTION
**Pull request by o-rumiantsev**
Closes #735

### CODE REVIEW

File: deploy/debian.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+RELEASE=$(lsb_release -sc)
```
Comment (belochub) [inline deploy/debian.sh]: Is this variable used anywhere?

Comment (o-rumiantsev) [reply deploy/debian.sh]: No, just forgot ti= remove it.

**Comment by tshemsedinov** [comment]
Landed in 6c58f97

**Comment by tshemsedinov** [comment]
I've landed this but note that `Closes` tag should be:
```
Closes: https://github.com/metarhia/impress/issues/735
```
not just `Closes #735`

**Pull request closed without merging**